plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1248,7 +1247,7 @@ func PodToleratesNodeTaints(pod *v1.Pod, meta interface{}, nodeInfo *schedulerca
 	return false, []algorithm.PredicateFailureReason{ErrTaintsTolerationsNotMatch}, nil
 }
 
-// Determine if a pod is scheduled with best-effort QoS
+// isPodBestEffort checks if pod is scheduled with best-effort QoS
```

Comment:
@gyliu513 even if this is not a public api, we'd better still follow the go style.